pkg/v1/cmd.go

```diff
@@ -178,7 +178,12 @@ func Run(ctx context.Context, logger *logrus.Logger, opts Options) error {
 				return fmt.Errorf("Failed to push changes.: %w", err)
 			}
 
-			var labelsToAdd []string
+			labelsToAdd := []string{
+				// The repos is set to use rebase merge method for making it easier to programmatically
+				// determine the commits which need to be carried. But the sync PR itself need to use merge.
+				// By adding this label we instruct tide to merge instead of using the default behaviour.
+				"tide/merge-method-merge",
```

Comment:
Should this be defined as `labels.TideMerge` or similar?

Reply:
@tmshort these constants come from the upstream `k8s.io/test-infra/prow/labels` and there is no constant defined for this label. Also - this label is configurable per-project so likely only exists in configs.

I added defined a constant for this at the top of the file to make it a bit better.